package/bleparser/almendo.py

```diff
@@ -0,0 +1,76 @@
+"""Parser for Almendo bluSensor BLE advertisements"""
+# black -l 79
+import logging
+from struct import unpack
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def parse_almendo(self, data, source_mac, rssi):
```

Comment:
```suggestion
def parse_blusensor(self, data, source_mac, rssi):
```

 I'm not sure about the name Almendo. If I check the website of the sensor, I only see BlueSensor, so why did you use Almendo? Even the URL is called blusensor.com. https://www.blusensor.com/en/products/